apisix/plugins/traffic-split.lua

```diff
@@ -203,6 +207,14 @@ local function new_rr_obj(weighted_upstreams)
         elseif upstream_obj.upstream then
             -- Add a virtual id field to uniquely identify the upstream key.
             upstream_obj.upstream.vid = i
```

Comment:
It seems the vid is no longer required?

Reply:
may not, here test vid:https://github.com/apache/apisix/blob/4dafab5afa3293b3d72007517246e01da385f8ef/t/plugin/traffic-split3.t#L398-L461